gbfs.md

```diff
@@ -96,13 +109,16 @@ Field Name          | Required  | Defines
 --------------------| ----------| ----------
 last_updated        | Yes       | Integer POSIX timestamp indicating the last time the data in this feed was updated
 ttl                 | Yes       | Integer representing the number of seconds before the data in this feed will be updated again (0 if the data should always be refreshed)
+version             | Yes       | String - GBFS version number to which the feed confirms, according to the versioning framework.
 data                | Yes       | JSON hash containing the data fields for this response
 
+
 Example:
 ```json
 {
   "last_updated": 1434054678,
   "ttl": 3600,
+  "version": "2.1",
```

Comment:
Just for clarity (because we're still at 1.0) I think all the example versions should be `1.0`.

Reply:
Makes sense. Updated. Since it's a small clarifying change, I think we can continue with the existing vote timeline (i.e. we don't need to call a re-vote).